qa/suites/fs/thrash/workloads/overrides/races.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      mds:
+        mds_sleep_rank_change: 5.0
```

Comment:
Hmmm... 5 microseconds is most likely not going to simulate any race IMO (the test does a config_set for 10 seconds).

Should this be at-least 1 second to catch races? (in which case we can just use sleep()...)

Reply:
Ya, that's an oops. Should be 5e6. Thanks.